bot.go

```diff
@@ -161,6 +161,33 @@ func DurationString(d time.Duration) string {
 	return strconv.Itoa(days) + "d" + rounded.String()
 }
 
+const smartQuotes = "“”"
```

Comment:
I would have used backtick, plus a comment

```suggestion
const smartQuotes = `“”` // not regular double quotes
```

Reply:
yes they are hard to differentiate on screen, good point (though hopefully the variable name
gives a hint)